addons/portal/static/src/scss/portal.scss

```diff
@@ -31,6 +31,10 @@ body {
     direction: ltr;
 }
 
+th{
```

Comment:
Let's remove these lines ;)
Tables design should be defined at BS level. Also this setting may mismatch user's preferences.

Anyway, if we want to customize Portal's tables only, next time we should target `.o_portal_my_doc_table th` specifically  ;)